gc_tests/run_miri_tests.rs

```diff
@@ -0,0 +1,78 @@
+use std::{
+    env,
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+use lang_tester::LangTester;
+use tempdir::TempDir;
+
+fn proj_dir() -> Box<Path> {
+    let mut p = PathBuf::new();
+    p.push(env::var("CARGO_MANIFEST_DIR").unwrap());
+    p.into_boxed_path()
+}
+
+fn test_dir() -> Box<Path> {
+    let mut p = PathBuf::new();
+    p.push(env::var("CARGO_MANIFEST_DIR").unwrap());
+    p.push("gc_tests");
+    p.push("tests");
+    p.push(".");
+    p.into_boxed_path()
+}
+
+fn main() {
+    let projd = proj_dir();
+    let testsd = test_dir();
+
+    let tmpd = TempDir::new("miri_tests").unwrap();
+    let mut binsd = PathBuf::new();
+    binsd.push(tmpd.path());
+    binsd.push("src");
+    binsd.push("bin");
+
+    let mut setup = Command::new("./gc_tests/miri_test_setup.sh")
```

Comment:
I wondered if it would be better to inline what `miri_test_setup.sh` does into Rust? But perhaps that's overkill.